test/helpers/kubectl.go

```diff
@@ -1722,40 +1760,12 @@ func (kub *Kubectl) CiliumPreFlightCheck() error {
 	// nodes cannot be show up yet, and the cilium-health can fail as a false positive.
 	var err error
 	body := func() bool {
-		switch GetCurrentIntegration() {
-		case CIIntegrationFlannel:
-		default:
-			err = kub.CiliumStatusPreFlightCheck()
-			if err != nil {
-				return false
-			}
-		}
-		err = kub.CiliumControllersPreFlightCheck()
-		if err != nil {
-			return false
-		}
-
-		switch GetCurrentIntegration() {
-		case CIIntegrationFlannel:
-		default:
-			err = kub.CiliumHealthPreFlightCheck()
-			if err != nil {
-				return false
-			}
-		}
-		err = kub.fillServiceCache()
-		if err != nil {
-			return false
-		}
-		err = kub.CiliumServicePreFlightCheck()
-		if err != nil {
-			return false
-		}
-		err = kub.ServicePreFlightCheck("kubernetes", "default")
-		if err != nil {
+		if err := kub.ciliumPreFlightCheck(); err != nil {
+			ginkgoext.By("Cilium is not ready yet: %s", err.Error())
```

Comment:
@eloycoto I totally understand where you are coming from. Right now, it was super difficult to try and debug failure here because we either display the same message whether a check is ok or not. On other checks, no message was printed but it would still print the message for checks that succeeded.

Super happy to figure out a better way. Overall it is incredibly useful to undertand why something is not ready yet. Correlating multiple logs, in particular the command output log is very difficult.

Reply:
@eloycoto I've changed it to only print errors if it is not a repeated error